src/utils/dependencies.utils.ts

```diff
@@ -29,7 +22,7 @@ export const destroyProviders = async (): Promise<void> => {
     await LifeCycleManager.triggerProviderDestroy(key);
   }
 
-  for (const key of Providers.getProviderInstancesByType('interceptor').keys()) {
+  for (const key of Providers.getProviderInstancesByType('microservice').keys()) {
```

Comment:
We don't need the call with `interceptor` anymore? 🤔

Reply:
Right, that was a leftover for sure.